tools/make-kani-release/src/main.rs

```diff
@@ -0,0 +1,174 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! This file is a glorified shell script for constructing a Kani release bundle.
+//! We use Rust here just to aid in making the "script" more robust.
+//!
+//! Run with `cargo run -p make-kani-release -- <version>` and this will produce
+//! (e.g.) `kani-1.0-x86_64-unknown-linux-gnu.tar.gz`.
+
+use std::{ffi::OsString, path::Path, process::Command};
+
+use anyhow::{bail, Result};
+
+fn main() -> Result<()> {
+    let version_string = parse_args()?;
+    let kani_string = format!("kani-{}", version_string);
+    let bundle_name = format!("{}-{}.tar.gz", kani_string, env!("TARGET"));
+
+    if !Path::new("src/kani-compiler").exists() {
+        bail!("Run from project root directory. Couldn't find 'src/kani-compiler'.");
+    }
+
+    let dir = Path::new(&kani_string);
```

Comment:
Can you make the installation path customizable?

Reply:
This is just building the release bundle. The installation path is anything (unpacked anywhere), nothing to configure here, so I'm not sure what you mean?

Reply:
I mean, generate the files somewhere different. For debug purpose, it might be nice to generate the bundle in a directory that is not the root folder of our repository.

Reply:
Not a blocker, but we should eventually do this.

For now, can you please ensure that our `.gitignore` will ignore the `tar.gz` file and the folder you are creating here.